infra/snd/01/nginx-ingress-values.yaml

```diff
@@ -2,13 +2,16 @@ apiVersion: helm.toolkit.fluxcd.io/v2beta1
 kind: HelmRelease
 metadata:
   name: nginx-ingress
-  namespace: config-cluster-flux
+  namespace: flux-config
 spec:
   chart:
     spec:
       version: 0.18.1
   values:
     controller:
+      service:
+        annotations:  
+          service.beta.kubernetes.io/azure-load-balancer-internal-subnet: "SNDADPNETSU1401" #TODO: Need to update once the platform values are available.
```

Comment:
@ppotturi - Are we leaving SNDADPNETSU1401 hard-coded for now?  We could use the postbuild substitution in the Cluster Bicep template, similar to how we've done it for the ASO secret values and App Config values.

Reply:
It will be changed to platform variables once they are available.

Reply:
okay cool.